app/controllers/counselor/groups_controller.rb

```diff
@@ -5,9 +5,10 @@ def index
   end
 
   def show
-    @current_group = Group.find(params[:id])
-    @counselors = @current_group.counselors
-    @kids = @current_group.kids
+    @group = Group.find(params[:id])
```

Comment:
This will allow counselors to view any group - whether part of their camp or not. not sure if we care at this point, but pointing it out.